projects/soft_999_eric_zhao_fw_103/src/main.c

```diff
@@ -0,0 +1,57 @@
+#include "adc.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+#include "wait.h"
+
+void prv_analog_read_a6(const GpioAddress *address, void *context) {
```

Comment:
Private functions should be declared static, ie `static void`. This means that it will only be able to be accessed in the file in which it is declared, and lets us know that it will only be called by other methods